halo/app/ready.go

```diff
@@ -0,0 +1,96 @@
+package app
+
+import (
+	"encoding/json"
+	"io"
+	"sync"
+
+	"github.com/omni-network/omni/lib/errors"
+)
+
+// Consist of all flags describing the current readiness status of a halo node.
+type readinessStatus struct {
+	// Indicates if the current halo instance uses an EVM RPC endpoint instead of running an execution client.
```

Comment:
nit:
```
// executionPeerAvailable defines whether a optional connection to geth HTTP RPC is available to query peers. It isn't available via the required Auth RPC (EngineAPI)
executionPeerAvailable bool
```

Reply:
I think you had the old version in the tab, I removed this flag because having a pointer gives me all the information I need and I don't need to maintain an additional flag.